northstar/src/main.rs

```diff
@@ -80,26 +80,26 @@ fn main() -> Result<(), Error> {
 }
 
 async fn run(config: Config) -> Result<(), Error> {
-    let mut runtime = runtime::Runtime::start(config)
+    let runtime = runtime::Runtime::start(config)
         .await
         .context("Failed to start runtime")?;
     let mut sigint = tokio::signal::unix::signal(SignalKind::interrupt())
         .context("Failed to install sigint handler")?;
     let mut sigterm = tokio::signal::unix::signal(SignalKind::terminate())
         .context("Failed to install sigterm handler")?;
 
-    let status = select! {
+    select! {
         _ = sigint.recv() => {
             info!("Received SIGINT. Stopping Northstar runtime");
-            runtime.stop_wait().await
+            runtime.stop()
         }
         _ = sigterm.recv() => {
             info!("Received SIGTERM. Stopping Northstar runtime");
-            runtime.stop_wait().await
+            runtime.stop()
         }
-        status = &mut runtime => status,
     };
-    match status {
+
+    match runtime.stop_wait().await {
```

Comment:
Why is this change needed? A `stop_wait` does the same as a `stop` and `stop_wait.await`.

Reply:
Hmmm yes, I think this is wrong :thinking: